sdk/resourcemanager/Azure.ResourceManager/src/Resources/Custom/TenantResource.cs

```diff
@@ -16,8 +16,10 @@ namespace Azure.ResourceManager.Resources
     [CodeGenSuppress("GetAsync", typeof(CancellationToken))]
     [CodeGenSuppress("GetAvailableLocations", typeof(CancellationToken))]
     [CodeGenSuppress("GetAvailableLocationsAsync", typeof(CancellationToken))]
+#pragma warning disable CA1825 // Avoid zero-length array allocations
```

Comment:
Curious: didn't @heaths do something to address this before?  I'm surprised to see it's back, but don't remember what the action was.

Reply:
Yes, it shouldn't be necessary anymore. Make sure your feature branch is rebased on main if you're seeing it. See https://github.com/Azure/autorest.csharp/issues/3261 for context.